src/test/java/racingcar/domain/RaceTest.java

```diff
@@ -0,0 +1,109 @@
+package racingcar.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import racingcar.config.RaceConfig;
+import racingcar.dto.CarMovementResponse;
+import racingcar.dto.WinnerResponse;
+
+@DisplayName("자동차 경주 테스트")
+class RaceTest {
+	private Car car1;
+	private Car car2;
+	private Car car3;
+	private List<Car> cars = new ArrayList<>();
+	private Race race;
+
+	@BeforeEach
+	void setUp() {
+		// given
+		car1 = new Car("pobi");
+		car2 = new Car("woni");
+		car3 = new Car("jun");
+
+		cars.add(car1);
+		cars.add(car2);
+		cars.add(car3);
+
+		race = new Race(cars);
+	}
+
+	@Test
+	@DisplayName("모든 자동차의 랜덤 숫자가 기준값 이상일 때, 모든 자동차가 이동한다.")
+	void Move_Each_Car_When_Random_Number_Greater_Than_Or_Equal_To_Condition_Number() {
+		// given
+		int randomNumber = RaceConfig.MOVE_CONDITION_NUMBER.getNumber();
+
+		// when
+		race.moveEachCars(randomNumber);
+
+		// then
+		assertThat(cars.stream()
+			.map(Car::getDistance)
+			.toList())
+			.isEqualTo(List.of(1, 1, 1));
```

Comment:
```suggestion
		// then
		assertThat(cars)
				.extracting(Car::getDistance)
				.containsExactly(1, 1, 1);
```

이렇게 하시면 더 깔끔해요. 전체적으로 적용해보시면 좋을 것 같아요1

Reply:
오... 테스트는 아직 많이 안 짜봐서 사실 JUnit 관련 문법이 많이 부족한 상태입니다... 이런 부분 너무 도움이 되네요!!